src/mlpack/tests/main_tests/preprocess_scale_test.cpp

```diff
@@ -18,14 +18,15 @@ static const std::string testName = "PreprocessScale";
 #include <mlpack/methods/preprocess/preprocess_scale_main.cpp>
 
 #include "test_helper.hpp"
-#include <boost/test/unit_test.hpp>
-#include "../test_tools.hpp"
+#include "../test_catch_tools.hpp"
+#include "../catch.hpp"
 
 using namespace mlpack;
 
 struct PreprocessScaleTestFixture
 {
  public:
+  static arma::mat dataset;
```

Comment:
I think it would be fine to leave this where it was before, unless there is some reason I overlooked to put it in the fixture. :+1:

Reply:
Not it was colliding with the dataset variable in image_load_test.cpp

Error was , dataset symbol aldeardy defined

Reply:
duplicate symbol '_dataset' in:
    CMakeFiles/mlpack_catch_test.dir/scaling_test.cpp.o
    CMakeFiles/mlpack_catch_test.dir/main_tests/preprocess_scale_test.cpp.o
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [bin/mlpack_catch_test] Error 1
make[2]: *** [src/mlpack/tests/CMakeFiles/mlpack_catch_test.dir/all] Error 2
make[1]: *** [src/mlpack/tests/CMakeFiles/mlpack_catch_test.dir/rule] Error 2
make: *** [mlpack_catch_test] Error 2

Reply:
Oh, I see.  Maybe we can just call it `scale_dataset` or something like that?

Reply:
is there any issue with declaring them as static ?

Reply:
Sure, I guess it is fine to leave it as-is.  I don't think it really makes a huge difference either way, I was just hoping to keep the diff as small as possible. So let's go with the way you have it now. :smile: